scripts/travis/imap-setup.sh

```diff
@@ -0,0 +1,186 @@
+#!/bin/sh -e
+
+#
+# ### This is a script to setup a dovecot imap server for testing rml_imap
+#
+
+#
+# Declare the important path variables
+#
+
+# Directories used in the system
+BASEDIR=$(git rev-parse --show-toplevel)
+BUILDDIR="${BASEDIR}/build/ci/imap"
+TRAVISDIR="${BASEDIR}/scripts/travis"
+RUNDIR="${BUILDDIR}/dovecot_run"
+TLSRUNDIR="${BUILDDIR}/dovecot_tls_run"
+CERTDIR="${BASEDIR}/raddb/certs/rsa"
+
+# The paths to the the configs for the different instances
+CONF="${BUILDDIR}/dovecot_conf/fr_dovecot.conf"
+TLSCONF="${BUILDDIR}/dovecot_conf/fr_tls_dovecot.conf"
```

Comment:
Use the LFS names here, i.e. `dovecot/etc/` for configuration files and `dovecot/log/` for logs.  Then it's easier to trash all the dovecot stuff at once and fiure out where things should live.  You also don't need the `fr_` prefix, it's in our build dir, we know we created it.